src/repl-frontend/scala/tools/nsc/interpreter/shell/ILoop.scala

```diff
@@ -183,11 +182,12 @@ class ILoop(config: ShellConfig, inOverride: BufferedReader = null,
 
   /** Standard commands **/
   lazy val standardCommands = List(
-    cmd("completions", "<string>", "output completions for the given string", completionsCommand),
-    cmd("edit", "<id>|<line>", "edit history", editCommand),
     cmd("help", "[command]", "print this summary or command-specific help", helpCommand),
-    historyCommand,
-    cmd("h?", "<string>", "search the history", searchHistory),
+    cmd("completions", "<string>", "output completions for the given string", completionsCommand),
+    // TODO maybe just drop these commands, as jline subsumes them -- before reenabling, finish scala.tools.nsc.interpreter.jline.HistoryAdaptor
+    //cmd("edit", "<id>|<line>", "edit history", editCommand),
+    //historyCommand,
```

Comment:
Current `emacs-sbt-mode` is integrated to the history command. It loads the history to emacs using the `:history` command and expects also commands executed through emacs to be there.

Reply:
@hvesalai thank, can you comment on https://github.com/scala/scala-dev/issues/690 ?